### ISSUE TITLE
Question about VoG on early training stage

### ISSUE THREAD

**Issue by Jasonlee1995** [chirag126/VOG#4]
While reading paper, I get confused about the conflicts between figure and the contents.

<img width="900" alt="image" src="https://user-images.githubusercontent.com/49643709/214746942-bfbc1229-9685-4811-b380-1c68c9bfd362.png">

Section 3.5, VoG understands early and late training dynamics part
- In the early training stage, samples having higher VoG scores have a lower average error rate as the gradient updates hinge on easy examples.
- This phenomenon reverses during the late-stage of the training, where, across all datasets, high VoG scores in the late-stage have the highest error rates as updates to the challenging examples dominate the computation of variance.

What I understand is below
- In early-stage training, easy examples have high VoG score
- In late-stage training, difficult examples have high VoG score

But Figure 2 seems different what I expected.

Early-stage training with highest VoG score doesn't look like easy examples, however lowest VoG score seems quite easy examples.

Is there something I'm missing?

**Comment by chirag-agarwall** [comment]
Thank you for reaching out and showing interest in our work. We would like to highlight that the images in Figure 2 are qualitative examples from the training set of the datasets, whereas the mean top-1 test set error plotted in Figures 5, 8, and 11 are shown for the testing set.
